styles/widgets/material/accordion.material.less

```diff
@@ -16,6 +16,7 @@
 
 
 .dx-accordion {
+    background-color: @accordion-background-color;
     color: @accordion-color;
     margin: 3px 3px 5px 3px;
```

Comment:
Is this worked as you want in a material theme? I mean we have margins to show shadows. Should we change it to paddings?

Reply:
I have updated dependencies. As for margins, I think we should test it and change in a context of an another issue if need